apps/fishing-map/features/reports/reports.hooks.ts

```diff
@@ -91,7 +90,7 @@ export function useFitAreaInViewport() {
 export const COARSE_SIMPLIFY_TOLERANCE = 0.1
 export const FINE_SIMPLIFY_TOLERANCE = 0.001
 
-export function getSimplificationByDataview(dataview: UrlDataviewInstance | Dataview) {
+export function getSimplificationByDataview(dataview: UrlDataviewInstance | Dataview | undefined) {
```

Comment:
I'd rollback this change to ensure there is always a Dataview

Reply:
👌🏼